deploy/renew_proxy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+credname=CREDNAME
+source /data/admin/wmagent/env.sh
+source /data/srv/wmagent/current/apps/wmagent/etc/profile.d/init.sh
+myproxy-get-delegation -v -l amaltaro -t 168 -s 'myproxy.cern.ch' -k $credname -n \
+  -o /data/certs/mynewproxy.pem && voms-proxy-init -rfc -voms cms:/cms/Role=production -valid 168:00 \
```

Comment:
In this line we are still using voms-proxy-init through the system paths as they are configured by default. Which in many cases (especially for the CERN machines) points to voms-client-java. [1]

[1]
```
[vocms0252]$ ll `which voms-proxy-init`
lrwxrwxrwx. 1 root root 33 Nov 17  2018 /usr/bin/voms-proxy-init -> /etc/alternatives/voms-proxy-init
[vocms0252]$ ll /etc/alternatives/voms-proxy-init
lrwxrwxrwx. 1 root root 25 Nov 17  2018 /etc/alternatives/voms-proxy-init -> /usr/bin/voms-proxy-init3
[vocms0252]$ cat /usr/bin/voms-proxy-init3
#!/bin/sh
VOMS_CLIENTS_JAVA_OPTIONS=${VOMS_CLIENTS_JAVA_OPTIONS:-"-XX:+UseSerialGC -Xmx16m"}
java $VOMS_CLIENTS_JAVA_OPTIONS -cp $(build-classpath voms-clients-java voms-api-java canl-java bcpkix bcprov commons-cli commons-io) org.italiangrid.voms.clients.VomsProxyInit "$@"
```